lib/strategy.js

```diff
@@ -47,6 +47,7 @@ function Strategy(options, verify) {
   options.sessionKey = options.sessionKey || 'oauth:twitter';
   options.tokenURL =
     options.tokenURL || 'https://api.twitter.com/2/oauth2/token';
+  options.scope = ['users.read', ...options.scope || []]
```

Comment:
```suggestion
  options.scope = Array.from(new Set(['users.read', ...options.scope || []]))
```

We should remove duplicities in scopes.